bin/.travis/init_solr.sh

```diff
@@ -3,7 +3,6 @@
 SOLR_PORT=${SOLR_PORT:-8983}
 SOLR_VERSION=${SOLR_VERSION:-4.10.4}
 DEBUG=${DEBUG:-false}
-SOLR_CONFS="eZ/Publish/Core/Search/Solr/Content/Resources/schema.xml"
```

Comment:
Moved in https://github.com/ezsystems/ezpublish-kernel/commit/d4d1a0be006f1087801a46527225f6b0034062c9 and https://github.com/ezsystems/ezplatform-solr-search-engine/commit/408fd5f03d19a369f158567562ada25eef98ddbe.